### PR TITLE
OCLOMRS-282: Update README to describe commands to use for SASS usage

### PR DESCRIPTION
**Pull request by abulojoshua1**
# JIRA TICKET NAME:
[Update README to describe commands to use for SASS usage](https://issues.openmrs.org/browse/OCLOMRS-282)

# Summary:
Recently, the app migrated from CSS to SASS usage. and the README should be updated to describe commands to use when you're using SASS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,13 @@ To run tests
  $ npm test -- -u --coverage
 ```
 
+### Creating a [Sass](https://sass-lang.com/) build
+```
+- To compile Sass to CSS, run the command below.
+
+$ npm run build:css && eslint ./src
+```
```
Comment (Duncanian) [inline README.md]: You can also add the command `npm run watch:css` to have automated sass compilation when one is working on a task.

Comment (abulojoshua1) [reply README.md]: Thanks, @Duncanian for the tip. I will surely add that.

File: README.md [hunk 2]
```diff
@@ -80,6 +80,13 @@ To run tests
...
+
```
[the remark this answers is not in the document]
Comment (abulojoshua1) [reply README.md]: @AGMETEOR , the command you are talking about seems to already be there, kindly take a closer look or be more specific on what you mean. I do not understand you yet on this subject.

File: README.md [hunk 3]
```diff
@@ -80,6 +80,18 @@ To run tests
  $ npm test -- -u --coverage
 ```
 
+### Creating a Sass build
+```
+- To compile Sass to CSS each time you make a change to a file with a .scss file
```
Comment (dkayiwa) [inline README.md]: Should it be Sass or SASS?

Comment (abulojoshua1) [reply README.md]: According to [this source](https://sass-lang.com/), they use `Sass` and not `SASS`.

Comment (dkayiwa) [reply README.md]: Then in that case, any reason why line 90 below uses SASS?

Comment (abulojoshua1) [reply README.md]: Thanks for the keep eye @dkayiwa, let me fix it in a moment.

**Pull request merged**